# Speed of the robot in m/s
float64 speed

# Delta is the turning angle of the servo. 
# Range should be -pi/4 to pi/4
float64 delta

# Bristle Roller Speed 
# Range should be -100 to 100
int32 brush_speed

# Conveyor Speed
# Range should be -100 to 100
int32 conveyor_speed